notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>Probably we need to clarify what mean "Make sure to have a Hugging Face account and login in" (at least add link where it should be loggined for improving usebility)</p><p>&gt;&gt;To generate a token, follow the link displayed in the cell output.</p><p>it is incorrect statement, login op not generate token, it requires to provide own generated token for authorization</p><p> </p><p>For more information on access tokens, refer to https://huggingface.co/docs/hub/security-tokens</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>fixed</p>